beacon_node/lighthouse_network/src/config.rs

```diff
@@ -449,20 +444,25 @@ pub fn gossipsub_config(network_load: u8, fork_context: Arc<ForkContext>) -> Gos
             }
         }
     }
-
+    let cloned_spec = spec.clone();
```

Comment:
you are being told by the compiler to do this because a couple lines below there is a `move`. However, cloning the entire spec is not necessary. Instead of creating a `cloned_spec`, create a variable with the value of `cloned_spec.message_domain_valid_snappy` here and use that variable below